src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_FIGURE_SIZE = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public int figureSize() {
+        return random.nextInt(MAX_FIGURE_SIZE);
+    }
+
+    public Figure whichFigure() {
```

Comment:
why did you choose this name? (in the task was`getRandomFigure`)

Reply:
sorry, I did not pay attention there was prepared name in task for that